test/e2e/serial/tests/configuration.go

```diff
@@ -123,11 +136,18 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			targetIdx, ok := e2efixture.PickNodeIndex(workers)
 			Expect(ok).To(BeTrue())
 			targetedNode := workers[targetIdx]
+			// node that will still be associated to initial mcps
+			initialMcpSampleNode := workers[len(workers)-targetIdx-1]
```

Comment:
can it be `targetNode`? what's the benefit of picking some other node?
are we guaranteed `len(workers)-targetIdx-1` is always in the interval `[0, len(workers))` ?

Reply:
> can it be targetNode? what's the benefit of picking some other node?

we chose targetNode for the new mcp. we need another sample node for the old mcps

> are we guaranteed len(workers)-targetIdx-1 is always in the interval [0, len(workers)) ?

good catch, let me add a filter

Reply:
>we chose targetNode for the new mcp. we need another sample node for the old mcps

Reply:
> I think we can make it work with just one node but is not too important

we must pass node to be examined for the desired vs current config. the node must be associated to the mcp that we are checking it's updates, if not the check would fail because we compare against the mcp.status.configuration.name. we can exclude the target node for the old mcp check, but we need eventually to work with a different node or node list (for now one node is sufficient maybe later we can extend the new function to check a node list as the tests require)